packages/Algernon.package/ALGCommentFilter.class/instance/filter.given..st

```diff
@@ -3,4 +3,6 @@ filter: anItemCollection given: aStringCollection
 
 	| commentKeyWord |
 	commentKeyWord := aStringCollection second.
-	^ anItemCollection select: [ :each | each matchesComment: commentKeyWord ]
\ No newline at end of file
+	^ (anItemCollection
+		select: [ :each | each matchesComment: commentKeyWord ])
+		do: [ :each | each fuzzyRank: self class classCommentFuzzyPenalty ]
```

Comment:
When a PR is this small, the comments also become more pedantic 😃: 
You can use `select:thenDo:` here